spec/lib/service_order_listener_spec.rb

```diff
@@ -10,21 +10,8 @@
       ManageIQ::API::Common::Request.with_request(request) { example.call }
     end
 
-    let(:message) { ManageIQ::Messaging::ReceivedMessage.new(nil, nil, payload, nil, client) }
-    let(:payload) { {"task_id" => "123", "state" => state} }
-    let!(:item) do
-      OrderItem.create!(
-        :count                       => 1,
-        :service_parameters          => "test",
-        :provider_control_parameters => "test",
-        :order                       => order,
-        :service_plan_ref            => "321",
-        :portfolio_item              => portfolio_item,
-        :topology_task_ref           => topology_task_ref
-      )
-    end
-    let(:order) { Order.create! }
-    let(:portfolio_item) { PortfolioItem.create!(:service_offering_ref => "321") }
+    let(:message) { double("ManageIQ::Messaging::ReceivedMessage") }
```

Comment:
Interesting that the specs are failing with the wrong number of Arguments error, even with a double here.